### PR TITLE
update release process doc

### PR DESCRIPTION
**Pull request by monopole**
an attempt to describe how we'll do releases in this three-module/one-repo world.

### CODE REVIEW

File: releasing/README.md [hunk 1]
```diff
@@ -4,44 +4,202 @@
...
+Executable files appear for various operating systems
+on the [release page].
+
+See the [installation instructions](../INSTALL.md)
+to perform an install.
+
+
+### sigs.k8s.io/kustomize
+
+This is the kustomize Go api.
```
Comment (Liujingfang1) [inline releasing/README.md]: api -> API

File: releasing/README.md [hunk 2]
```diff
@@ -4,44 +4,202 @@
...
+There's only one package in this module.  It's called `main`,
+and it holds the _kustomize_ executable.
+
+
+#### Release artifacts
+
+Executable files appear for various operating systems
+on the [release page].
+
+See the [installation instructions](../INSTALL.md)
```
Comment (Liujingfang1) [inline releasing/README.md]: This file should (../docs/INSTALL.md)

File: releasing/README.md [hunk 3]
```diff
@@ -4,44 +4,202 @@
...
+require sigs.k8s.io/kustomize/v4 v4.0.1
 ```
 
-## Do a real (cloud) release
+#### Release artifacts
+
+Source code tar balls and zip files (no executables)
+appear on the [release page].
+
+### sigs.k8s.io/kustomize/pluginator
```
Comment (Liujingfang1) [inline releasing/README.md]: Can you also explain the policy of when to release a new version of pluginator and when to update the pluginator in the api module?

File: releasing/README.md [hunk 4]
```diff
@@ -4,44 +4,202 @@
...
+
+#### Packages
+
+There's only one package in this module.  It's called `main`,
+and it holds the _kustomize_ executable.
+
+
+#### Release artifacts
+
+Executable files appear for various operating systems
```
Comment (Liujingfang1) [inline releasing/README.md]: Add the name of release artifacts like kustomize/v3.2.1

File: releasing/README.md [hunk 5]
```diff
@@ -4,44 +4,202 @@
...
 
 ```
-./releasing/localbuild.sh
+require sigs.k8s.io/kustomize/v4 v4.0.1
 ```
 
-## Do a real (cloud) release
+#### Release artifacts
+
+Source code tar balls and zip files (no executables)
```
Comment (Liujingfang1) [inline releasing/README.md]: add the name of release artifact like v3.2.1

File: releasing/README.md [hunk 6]
```diff
@@ -4,44 +4,202 @@
...
+There's only one package in this module.  It's called `main`,
+and it holds the _pluginator_ executable.
+
+At the time of writing this binary is only of interest to
+someone writing a new builtin transformer or generator.
+See the [plugin documentation](../docs/plugins).
+
+#### Release artifacts
+
+Executables appear on the [release page].
```
Comment (Liujingfang1) [inline releasing/README.md]: add the name of release artifact like pluginator/v1.0.0

File: releasing/README.md (hunk 3 above)
Comment (monopole) [reply releasing/README.md]: done

File: releasing/README.md (hunk 5 above)
Comment (monopole) [reply releasing/README.md]: done

File: releasing/README.md (hunk 4 above)
Comment (monopole) [reply releasing/README.md]: done

File: releasing/README.md (hunk 6 above)
Comment (monopole) [reply releasing/README.md]: done

**Pull request merged**